### ISSUE TITLE
WebNowPlaying not working, but other plugins are.

### ISSUE THREAD

**Issue by C2N14** [tjhrulz/WebNowPlaying#10]
I don't know what happened, I don't recall updating anything recently but now it seems like it's not working:
![Screenshot 1](https://user-images.githubusercontent.com/29585778/35478207-2f09349e-0394-11e8-9fb5-94b112313527.png)

Neither WebNowPlayingExample or Cleartext seem to work, but if I start playing spotify, Cleartext seems to pick it up:
![Screenshot 2](https://user-images.githubusercontent.com/29585778/35478623-59376712-039f-11e8-8c17-ce3ac163b3ba.png)

As far as I know everything is installed and activated, and Cleartext is configured correctly to use WebNowPlaying as a secondary source and it worked fine before, if I need to provide any log files please let me know.

**Comment by C2N14** [comment]
Oh god I feel like such an idiot. I DID update Firefox a little ago and it seems `network.websocket.allowInsecureFromHTTPS` went back to `false`. Switched it to `true` and now it works like a charm .-.
Sorry for wasting your time.

**Comment by tjhrulz** [comment]
You beat me by 9 seconds

**Comment by C2N14** [comment]
About Firefox says it is 58.0 but I was using a quite older version before updating (something like 55.x or 56.x). I think when I initially updated I was having an issue with a setting I changed about the browser before updating so I decided to do a clean install and manually install all the Add-On's, but it never came to mind that I needed to do that extra step before it could work.
So I think it's safe to say it's got nothing to do with Firefox or the Add-On, just me forgetting things 😓 

Anyways, thanks for your amazing work, I'm very looking forward to that Twitch Overlay! 👍
